pkg/webhook/admission/http.go

```diff
@@ -26,15 +26,9 @@ import (
 	"net/http"
 
 	"k8s.io/api/admission/v1beta1"
-	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/serializer"
 )
 
```

Comment:
this is actually fine as a distinct scheme, IIRC, since it's being used to encode/decode only the admissionrequest.  You can follow what the example webhook does in the kube tests -- just make sure to add the appropriate types like it does in `scheme.go`.